addon/components/hyper-table-v2/manage-columns.hbs

```diff
@@ -32,10 +32,12 @@
             </div>
           {{/if}}
           {{#each fields as |field|}}
-            <div class="field" {{on "click" (fn this.columnVisibilityUpdate field)}}>
-              <UpfCheckbox
-                @value={{field.visible}} @size="sm"
-                data-control-name={{concat "column_definition_toggle_checkbox_" field.definition.key}} />
+            <div class="field" role="button" {{on "click" (fn this.columnVisibilityUpdate field)}}>
+              {{#if field.isLoading}}
+                <div><i class="fa fa-spinner fa-spin"></i></div>
```

Comment:
Do you really need to wrap `i` in `div`?

Reply:
Yes, otherwise the center spinning point is a bit off - it does not render properly